src/jit/lclvars.cpp

```diff
@@ -7307,3 +7241,111 @@ void Compiler::lvaDispVarSet(VARSET_VALARG_TP set, VARSET_VALARG_TP allVars)
 }
 
 #endif // DEBUG
+
+LclVarTable::LclVarTable()
+    : m_count(0), m_capacity(0), m_array(nullptr)
+{
+}
+
+LclVarTable::LclVarTable(Compiler& compiler, unsigned initialSize)
+    : m_count(0), m_capacity(0), m_array(nullptr)
+{
+    Grow(compiler, initialSize < 8 ? 16 : initialSize * 2);
+    m_count = initialSize;
+}
+
+LclVarTable& LclVarTable::operator=(const LclVarTable& other)
+{
+    m_count = other.m_count;
+    m_capacity = other.m_capacity;
+    m_array = other.m_array;
+
```

Comment:
Yes, believe it or not--we share the lclVar table between a root compiler and its inlinee compilers.